src/input/SpaceNavInputDriver.h

```diff
@@ -38,9 +38,11 @@ class SpaceNavInputDriver : public InputDriver
     void run() override;
     bool open() override;
     void close() override;
+    void setDominantAxisOnly(bool var);
 
     const std::string & get_name() const override;
 
 private:
     bool spnav_input();
+    bool DominantAxisOnly=true;
```

Comment:
Member variables should start with a lower-case character. Also, uniform initialization is preferable:
`bool dominantAxisOnly{true};`